bin/serve.js

```diff
@@ -4,6 +4,7 @@
 const path = require('path')
 
 // Packages
+const https = require('https')
```

Comment:
This is not a package - it needs to be part of the "Native" import group.

Reply:
Sure thing; force of habit to add to the end and glossed over the comments :slightly_smiling_face: